languages/bash/reference/README.md

```diff
@@ -0,0 +1,60 @@
+# Bash Reference
+
+## Types
+
+- [string][type-string]
+- [float][type-float]
+- [integer][type-int]
+- [double][type-double]
+- [boolean][type-boolean]
+- [regular expression][type-regex]
+- [function][type-function]
+- [array][type-array]
+- [associative array][type-map]
+
+[type-array]: ../../../reference/types/array.md
+[type-boolean]: ../../../reference/types/boolean.md
+[type-double]: ../../../reference/types/double.md
+[type-float]: ../../../reference/types/floating_point_number.md
+[type-function]: ../../../reference/types/function.md
+[type-int]: ../../../reference/types/integer.md
+[type-map]: ../../../reference/types/map.md
+[type-regex]: ../../../reference/types/regular_expression.md
+[type-string]: ../../../reference/types/string.md
+
```

Comment:
I believe that these are generally instructive to how to design what this documentation links to... so I feel like this is currently "placeholder" material until the actual content is written specific to Bash... can you confirm?

Reply:
Well, tracks don't _have_ to write track-specific documents describing concepts. For a starter document I think it is fine to link to the general concept documents. At some point, these links can then be changed to link to github issues or the implemented exercise.